### PR TITLE
Node red mail usetls

### PR DESCRIPTION
**Pull request by Schreck1985**
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Proposed changes
Have the option to disable TLS checking for sending mails. See (https://discourse.nodered.org/t/node-red-node-email-tls-check/1853)

## Checklist

- [x] I have read the [contribution guidelines](https://github.com/node-red/node-red-nodes/blob/master/CONTRIBUTING.md)
- [x] For non-bugfix PRs, I have discussed this change on the mailing list/slack team.
- [ ] I have run `grunt` to verify the unit tests pass
- [ ] I have added suitable unit tests to cover the new/changed functionality

### CODE REVIEW

File: social/email/61-email.js [hunk 1]
```diff
@@ -55,7 +56,8 @@ module.exports = function(RED) {
         var smtpOptions = {
             host: node.outserver,
             port: node.outport,
-            secure: node.secure
+            secure: node.secure,
+            tls: {rejectUnauthorized: node.tls}
```
Comment (dceejay) [inline social/email/61-email.js]: just checking - letting this be null or undefined is OK ?

Comment (Schreck1985) [reply social/email/61-email.js]: Maybe we should check for null or at least set default value of true. Sorry, it is my very first touch to JavaScript.

Comment (dceejay) [reply social/email/61-email.js]: Yes - (hey no problem, glad you are learning). Yes - further up set it true by default - then test if n.tls === false - so only set it false if explicitly set to false by the user.

**Comment by Schreck1985** [comment]
Damn, I don't know what is wrong with mit GIT config. Sorry for that.

**Comment by dceejay** [comment]
Thanks for persevering

**Pull request merged**